external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSource.scala

```diff
@@ -81,14 +81,16 @@ import org.apache.spark.util.UninterruptibleThread
  * To avoid this issue, you should make sure stopping the query before stopping the Kafka brokers
  * and not use wrong broker addresses.
  */
-private[kafka010] case class KafkaSource(
```

Comment:
the unique id of the object? maybe. I am not sure. if the same kafkaDF is used twice in a query (say, union two duplicate the data), will it actually create two distinct source objects, or reuse the same one. And in that case do we want them to match or not?